recipes/logic1/recipe.yaml

```diff
@@ -0,0 +1,60 @@
+context:
+  version: 0.1.0
+  python_min: "3.11"
+
+package:
+  name: logic1
+  version: ${{ version }}
+
+source:
+  url: https://github.com/logic1-eu/logic1/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: 7149ce98cc9381da14de26f871c00f14fef12d0b3e482bef4c555ccfe8cc558d
+
+build:
+  number: 0
+  # Cannot be noarch, since we want to skip Windows,
+  # because transitive dependency `bdw-gc` is not available for Windows.
+  #noarch: python
```

Comment:
I would turn this back to `noarch: python`. Typically if a dependency (pytorch is a common example), is not available on windows through conda-forge, we just ignore the windows CI failure since there is nothing inherently platform specific about the package, and windows users are left to get the dependency installed themselves.

Reply:
Done in d453814597.